### PR TITLE
Whelen Dominator Series

### LINKED ISSUE #113: Whelen Dominator Series
## Feature
All of the Whelen Dominators (2, 4, 6, 8)

## Use Case
Well i believe the Dominator Series from Whelen is very popular across police agencies and i think lots of people would use them

## Context
dom-plus-grp

### PR DESCRIPTION
**Pull request by SentryGunMan**
Fixes #113

### CODE REVIEW

File: materials/sentry/props/dominator/dom.vmt [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    $phong "1"
+    $phongexponent "7"
+    $phongboost "1"
+    $phongfresnelranges "[.05 .3 1]"
+	"$carpenisenabled" "1"
+	Proxies // proxies are listed inside this block
+	{
+		Equals // a proxy which produces a penis
+		{
+			srcVar1 $carpenisenabled
```
Comment (JoshPiper) [inline materials/sentry/props/dominator/dom.vmt]: ?

File: lua/autorun/photon/library/auto/sgm_2dominator.lua [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+local COMPONENT = {}
+
+COMPONENT.Model = "models/sentry/props/dominator2.mdl"
+COMPONENT.Lightbar = true
+COMPONENT.Skin = 0
+COMPONENT.Category = "Lightbar" 
+COMPONENT.Bodygroups = {}
+COMPONENT.DefaultColors = {
+    [1] = "RED",
+	[2] = "BLUE",
```
Comment (creator2013) [inline lua/autorun/photon/library/auto/sgm_2dominator.lua:15]: imma scream about that indent

File: lua/autorun/photon/library/auto/sgm_2dominator.lua [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+			Vector( 1, 0, 0 ),
+		}
+	},
+}
+
+COMPONENT.Positions = {
+
+	[1] = { Vector( -0.2, -2.14, 0.01 ), Angle( 0, 0, 0 ), "dominator" },
+	[2] = { Vector( -0.2, 2.14, 0.01 ), Angle( 0, 0, 0 ), "dominator" },
+	--Vector( -14.75, 0.2, 0.01 ), Angle( 0, 0, 0 )
```
Comment (creator2013) [inline lua/autorun/photon/library/auto/sgm_2dominator.lua]: Is this still needed?

File: lua/autorun/photon/library/auto/sgm_2dominator.lua [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+COMPONENT.Sections = {
+	["auto_dom"] = {
+        [1] = {
```
Comment (creator2013) [inline lua/autorun/photon/library/auto/sgm_2dominator.lua]: Indent :(

**Review by SuperMighty1 — APPROVED** [review]
sheeesh

**Pull request merged**